ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/ShelleyHFC.hs

```diff
@@ -405,17 +405,18 @@ instance (SOP.All ShelleyBasedEra eras, Typeable eras) => ToCBOR (ShelleyTxOut e
     where
       each ::
            ShelleyBasedEra era
-        => SOP.Index eras era03
+        => SOP.Index eras era
```

Comment:
I must have fat-fingered the `03` when trying to line jump in `emacs` or something 😲